docs/components/dartpad/chips/choice/main.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(App());
+}
+
+class App extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Choice Chips Demo',
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        appBar: AppBar(
+          title: Text('Choice Chips Demo'),
+        ),
+        body: Center(
+          child: Column(
+            children: [
+              ChoiceChip(
```

Comment:
The colors in the screenshot of the choice chips seem off, can you look at https://gallery.flutter.dev/#/demo/chip and update the demo and screenshot to match that.

Reply:
Done